### DISCUSSION TITLE
Plugin Error

### DISCUSSION THREAD

**Discussion by ghost** [Eric-pm-pl/MultiPlayerCounter#4] [Q&A] [answered]
I Need your help, pli
![Captura de pantalla 2022-04-23 053249](https://user-images.githubusercontent.com/95693697/164888966-3e77ebd8-ac62-4e9a-9a65-301da4322eee.png)
s

**Comment by 0xliam627** [comment, marked as answer]
> I Need your help, pli ![Captura de pantalla 2022-04-23 053249](https://user-images.githubusercontent.com/95693697/164888966-3e77ebd8-ac62-4e9a-9a65-301da4322eee.png) s

You need to download [pmquerylib ](https://github.com/jasonwynn10/libpmquery) virion for the plugin to run

**Reply by ghost** [reply]
Thank you.
But I have another problem.
How do I install it?

**Reply by 0xliam627** [reply]
> Thank you. But I have another problem. How do I install it?

1. Download it
2. Download the [DEVirion ](https://poggit.pmmp.io/p/DEVirion/1.2.8) plugin
3. Extract it to the folder YourServer/virions
4. Run server

**Reply by ghost** [reply]
Perfect, thank you very much
